pkg/ebpf/bytecode/runtime/runtime_asset.go

```diff
@@ -40,6 +43,11 @@ type CompiledOutput interface {
 type RuntimeAsset struct {
 	filename string
 	hash     string
+
+	// Telemetry
+	compilationSuccess  int64
+	compilationFailure  int64
```

Comment:
Is there a difference between `compilationFailure = 0` and `compilationSuccess = 1` and vice versa?

Reply:
In a way, yes, since `compilationSuccess = 0` could either mean that compilation wasn't enabled, or it was enabled but it failed, whereas `compilationFailure = 1` explicitly states that compilation was enabled but failed (and vice versa). 
Regardless, I switched to using an enum to describe the result of the compilation, so these fields have been merged into 1